### PR TITLE
Da001 add competency table

### PR DESCRIPTION
**Pull request by MartaMartaZielonka**

### CODE REVIEW

File: src/databaseScripts/InsertData.sql [hunk 1]
```diff
@@ -14,6 +14,225 @@ INSERT INTO band(band_level, band_name)
 	(5, "Associate"),
 	(6, "Apprentice"),
 	(6, "Trainee");
+select * from band;
```
Comment (EwaRamlo) [inline src/databaseScripts/InsertData.sql]: delete select statement

**Review by EwaRamlo — CHANGES_REQUESTED** [review]

File: src/databaseScripts/InsertData.sql (hunk 1 above)
Comment (JohnWalker-Kainos) [inline src/databaseScripts/InsertData.sql]: Do you need the select in here?

Comment (MartaMartaZielonka) [reply src/databaseScripts/InsertData.sql]: done

**Review by JohnWalker-Kainos — CHANGES_REQUESTED** [review]
I'm happy enough with that... other than the comment about the random select statement at the end of the band insertion lines.

**Review by EwaRamlo — APPROVED** [review]

File: src/databaseScripts/InsertData.sql (hunk 1 above)
Comment (MartaMartaZielonka) [reply src/databaseScripts/InsertData.sql]: You're right, that is not needed, I deleted it.

**Review by JohnWalker-Kainos — APPROVED** [review]

**Pull request merged**